### PR TITLE
budget api wire up

### PR DESCRIPTION
**Pull request by BenjaminGroseclose**
Initial work for #3

### CODE REVIEW

File: server/budgettracker.business/Api/BudgetApi.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        public BudgetApi(IBudgetRepository budgetRepository)
+        {
+            _budgetRepository = budgetRepository;
+        }
+
+        public async Task<ApiResponse> CreateBudget(ApiRequest request)
+        {
+            // Authenticate the User here
+
+            // Convert request buget to Common Budget model
```
Comment (ianbro) [inline server/budgettracker.business/Api/BudgetApi.cs]: Unnecessary comments

File: server/budgettracker.data/Repositories/Interfaces/IBudgetRepository.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+using System.Threading.Tasks;
+using budgettracker.common.Models;
+
+namespace budgettracker.data.Repositories.Interfaces
+{
+    public interface IBudgetRepository
+    {
+        /// <summary>
```
Comment (ianbro) [inline server/budgettracker.data/Repositories/Interfaces/IBudgetRepository.cs]: Great comments!

File: server/budgettracker.business/Api/BudgetApi.cs (hunk 1 above)
Comment (BenjaminGroseclose) [reply server/budgettracker.business/Api/BudgetApi.cs]: Just place holder to the wire up.

Comment (ianbro) [reply server/budgettracker.business/Api/BudgetApi.cs]: oh ok

**Pull request merged**